tests/integration/ceph_base_object_test.go

```diff
@@ -142,6 +92,74 @@ func runObjectE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite
 	assert.True(s.T(), k8sh.CheckPodCountAndState("rook-ceph-mgr", namespace, 1, "Running"))
 	logger.Infof("Ceph MGRs are running alright")
 
+	/* Testing creation/deletion of objects using Object Bucket Claim */
+	logger.Infof("Step 3 : Create Object Bucket Claim with reclaim policy delete")
+
+	cobErr := helper.BucketClient.CreateBucketStorageClass(namespace, storeName, storageClassName, "Delete", region)
+	require.Nil(s.T(), cobErr)
+	cobcErr := helper.BucketClient.CreateObc(obcName, storageClassName, bucketname, true)
+	require.Nil(s.T(), cobcErr)
+
+	logger.Infof("Checking to see if the obc,secret,cm has been created")
+	for i = 0; i < 4 && helper.BucketClient.ObcExists(obcName) == false; i++ {
+		logger.Infof("(%d) obc check sleeping for 5 seconds ...", i)
+		time.Sleep(5 * time.Second)
+	}
+	assert.True(s.T(), helper.BucketClient.ObcExists(obcName))
```

Comment:
more efficient to have a loop counter above and just check if the loop ended because the obc (and cm, secret) all exist, or ended due to max iterations.

Reply:
resolved